src/Directory.Build.props

```diff
@@ -20,6 +20,8 @@
     <PlatformToolset>v142</PlatformToolset>
     <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
     <AssetTargetFallback>netcoreapp5.0</AssetTargetFallback>
+    <IsTargetFrameworkNet5OrGreater>false</IsTargetFrameworkNet5OrGreater>
+    <IsTargetFrameworkNet5OrGreater Condition="'$(TargetFramework)' == 'net5.0' Or '$(TargetFramework)' == 'net6.0'">true</IsTargetFrameworkNet5OrGreater>
```

Comment:
If the TFM has an sdk projection, e.g. net6.0-windows10.0.18362.0, do we still have this property set to true?